app/customer.py

```diff
@@ -0,0 +1,28 @@
+import dataclasses
+import math
+from typing import Any
+
+
+FUEL_PRICE = 2.4
```

Comment:
Why it is constant? I can place any variable in JSON file and I want use it not constant in code